lib/heartcheck/checks/webservice.rb

```diff
@@ -12,6 +12,17 @@ def validate
         end
       end
 
+      def uri_info
```

Comment:
What about adding a comment to say this method is necessary for the `/inspect` endpoint?